### PR TITLE
docs: VlsChecker is just named export

### PR DESCRIPTION
**Pull request by angus96**
fix: VlsChecker is just named export, just update README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ npm i vite-plugin-checker-vls -D
 
 ```js
 import Checker from 'vite-plugin-checker'
-import VlsChecker from 'vite-plugin-checker-vls'
+import {VlsChecker} from 'vite-plugin-checker-vls'
```
Comment (fi3ework) [inline README.md:68]:
```suggestion
import { VlsChecker } from 'vite-plugin-checker-vls'
```

File: README.md [hunk 2]
```diff
@@ -133,7 +133,7 @@ e.g.
...
```
Comment (fi3ework) [inline README.md:136]:
```suggestion
import { VlsChecker } from 'vite-plugin-checker-vls'
```

**Review by fi3ework — CHANGES_REQUESTED** [review]
We could have a better format and the commit message should be changed to `docs: xxx` as this is only a documentation-related fix.

File: README.md (hunk 1 above)
Comment (angus96) [reply README.md:68]: lgtm, it seems we have different lint style😂

**Comment by fi3ework** [comment]
Thanks! @angus96

**Pull request merged**